java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java

```diff
@@ -217,9 +231,9 @@ public void computeResult_callsNoiseCorrectly() {
   @Test
   public void computeResult_addsNoiseToSum() {
     // Mock the noise mechanism so that it adds noise to the sum == 10.0.
-    mockDoubleNoise(noise, 10);
+    mockDoubleNoise(10);
     // Mock the noise mechanism so that it adds noise to the count == 0.
-    mockLongNoise(noise, 0);
+    mockLongNoise(0);
```

Comment:
is it necessary? It should be 0 since it was in a setup method. IIUC setup is called every time before executing each test

Reply:
This is not something I added, It was already in tests. I merely changed it to remove the noise parameter. 

Aside from that, I feel from readability point if you are using mockDoubleNoise(...) with a different value than default, you should also specify in the same test what mockLongNoise(..) value should be even if it is the same. You won't have to look back up in the setup method to know what the default value is but that is just my personal opinion.

Reply:
not sure, we can say the same when specifying , e.f, upper and lower bounds but leave other params out of context, because only these value matter in this tests, everything else is not improtant.

Reply:
but you can keep it since it was before, we can remove it later if needed